barretenberg/cpp/src/barretenberg/vm/avm/generated/circuit_builder.cpp

```diff
@@ -19,673 +20,687 @@ AvmCircuitBuilder::ProverPolynomials AvmCircuitBuilder::compute_polynomials() co
     ProverPolynomials polys;
 
     // Allocate mem for each column
-
-    for (auto& poly : polys.get_to_be_shifted()) {
-        poly = Polynomial{ /*memory size*/ num_rows - 1,
-                           /*largest possible index*/ num_rows,
-                           /*make shiftable with offset*/ 1 };
-    }
+    AVM_TRACK_TIME("circuit_builder/init_polys_to_be_shifted", ({
```

Comment:
We have a similar time tracking mechanism that ties into google benchmark, BB_OP_COUNT_TIME. It will report as a google benchmark counter

Reply:
Thanks, will look into it!